packages/jellyfish-transaction/src/script/defi/dftx_pool.ts

```diff
@@ -94,7 +94,7 @@ export interface PoolRemoveLiquidity {
  */
 export class CPoolRemoveLiquidity extends ComposableBuffer<PoolRemoveLiquidity> {
   static OP_CODE = 0x72
-  static OP_NAME = 'DEFI_OP_POOL_REMOVE_LIQUIDITY'
+  static OP_NAME = 'OP_DEFI_TX_POOL_REMOVE_LIQUIDITY'
```

Comment:
So implementation OP_NAME is the same as mapping.ts OP_NAME now?

Reply:
Yeah, I think it's better for consistency, seeing as 3 people commented on it with varying conclusion.